pkg/compose/layers.go

```diff
@@ -288,7 +288,7 @@ func PostAppLayersManifests(ctx context.Context, appRef string, layers map[strin
 			fmt.Printf("  |-> skipping layer manifest publishing for dryrun\n")
 		} else {
 			fmt.Printf("  |-> posting a layer manifest for architecture: %s...", arch)
-			digest, err := manSvc.Put(ctx, manifest)
+			digest, err := manSvc.Put(ctx, manifest, distribution.WithTag(desc.Digest.Hex()))
```

Comment:
maybe we should give it a useful tag. The UI code might need some type of naming convention to know to ignore these.

Reply:
I think it is a good idea. Maybe the following tag naming convention will work `layers-<arch>-<short-hash>`?

Reply:
sounds good to me.

Reply:
done